include/crm/pengine/status.h

```diff
@@ -283,6 +283,7 @@ struct resource_s {
 
     GHashTable *meta;
     GHashTable *parameters;
+    xmlNode *versioned_parameters;
```

Comment:
Anything under include/crm without "internal" in its filename is considered the public API. To preserve backward compatibility in the public ABI, any new structure members must be added as the last member at the end of the structure. Unfortunately, it can make the code less organized/readable, but it's the only way to avoid requiring that all client applications be recompiled/linked.